orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -694,12 +698,12 @@ def __init__(self, parent: OWWidget, select=SELECTNONE):
         self.clear_data()
         self.subset = None  # current subset input, an array of indices
         self.subset_indices = None  # boolean index array with indices in self.data
-
+        self.start_point = None  # automatic position for adding single lines in peak_apply
+        self.peak_state = 0  # Value for determining what our peak clear should do
```

Comment:
Why does `peak_state` do? What do its values mean? It is not clear from the code why it is needed, I'd guess it is really not. Peak reappearance should be handled differently.